addon/components/time-series-chart.js

```diff
@@ -361,18 +394,30 @@ const TimeSeriesChartComponent = ChartComponent.extend(LegendMixin,
 
   // If there is a dynamic x axis, then assume the value that it is given,
   // and if it is not a dynamic x axis, set it to the number of x axis ticks.
+  //
   // For a dynamic x axis, let the max number of labels be the minimum of
   // the number of x ticks and the assigned value. This is to prevent
   // the assigned value from being so large that labels flood the x axis.
-  maxNumberOfLabels: Ember.computed('numXTicks', 'dynamicXAxis', function(key, value){
+  maxNumberOfLabels: Ember.computed('numXTicks', 'dynamicXAxis', 'maxNumberOfRotatedLabels', 'xAxisVertLabels', function(key, value){
+    var allowableTicks = this.get('xAxisVertLabels') ? this.get('maxNumberOfRotatedLabels') : this.get('numXTicks');
+
```

Comment:
I complained about `?:` elsewhere in the previous iteration of this file, but my concern about it is general.

Up to you whether you want to change the `?:` to an `if () { } else { }` here too. I think the latter is more readable, partly because it's on multiple lines, and more importantly because the branch is more obvious.